### ISSUE TITLE
pipewire: no jack input / output available

### ISSUE THREAD

**Issue by ycollet** [rncbc/qtractor#424]
I am working on Fedora 38 with pipewire 1.0.0 and qtractor 0.9.34.
When I start qtractor; there are no inputs / outputs displayed in the session manager.
A "working" icon is displayed and when I try to quit qtractor, the GUI disappears but a background process is still here and only a kill -9 can close this one.
I used strace on qtractor and I've got a lot of message like this one:
```
poll([{fd=4, events=POLLIN}, {fd=8, events=POLLIN}, {fd=10, events=POLLIN}, {fd=12, events=POLLIN}, {fd=15, events=POLLIN}, {fd=16, events=POLLIN}, {fd=45, events=POLLIN}], 7, 0) = 1 ([{fd=45, revents=POLLIN}])
sched_yield()                           = 0
```

**Comment by rncbc** [comment]
recently fixed in git master (qtractor >= 0.9.36.25git.[d2d9fa](https://github.com/rncbc/qtractor/commit/d2d9fa))

apparently pipewire 1.0.0 is to blame; 0.3.85 was fine; affets all qtractor releases to date.

**Comment by ycollet** [comment]
Tested and it's fixed.
Thanks again. I close the ticket.
